common/src/chain/block/block_index.rs

```diff
@@ -12,6 +12,7 @@ use super::timestamp::BlockTimestamp;
 pub struct BlockIndex {
     block_id: Id<Block>,
     block_header: BlockHeader,
+    skip: Option<Id<Block>>,
```

Comment:
When is this `None`? Is it only in genesis or are there other cases?

Reply:
Just genesis